src/admin/users/usersTable.component.tsx

```diff
@@ -196,10 +199,38 @@ function UsersTable() {
               </MenuItem>,
             ]
           : []),
+        <MenuItem
+          key="delete"
+          aria-label={`Delete user ${row.original.username}`}
+          onClick={() => {
+            setRequestType('delete');
+            setSelectedUser(row.original);
+            closeMenu();
+          }}
+          sx={{ m: 0 }}
+        >
+          <ListItemIcon>
+            <DeleteIcon />
+          </ListItemIcon>
+          <ListItemText>Delete</ListItemText>
+        </MenuItem>,
       ];
     },
   });
-  return <MaterialReactTable table={table} />;
+  return (
+    <>
+      <MaterialReactTable table={table} />
+      {selectedUser && (
+        <DeleteUserDialogue
+          open={requestType === 'delete'}
+          onClose={() => {
+            setRequestType(false);
+          }}
+          selectedUser={selectedUser}
+        />
+      )}
+    </>
```

Comment:
Is there a reason this dialogue is rendered differently than the other dialogue?

Reply:
The add and edit dialog is currently be rendering using the "renderCreateRowDialogContent" prop that is in `useMaterialReactTable` function. There isn't a equivalent for delete so it has to be done outside `useMaterialReactTable` function.